### PR TITLE
fix: triggers use correct consumer group id generated from template

### PR DESCRIPTION
**Pull request by Cali0707**
Currently, the triggers will always use a consumer group id generated from the default template. Instead, they should use the most up to date version of the template.

## Proposed Changes

- Add a rekt test to test for regressions here
- Watch the configmap for changes

**Release Note**

```release-note
:bug: triggers now correctly generate the consumer group id from the template, rather than just the default template
```

**Docs**

### CODE REVIEW

File: test/rekt/resources/configmap/configkafkafeatures/configKafkaFeatures.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	}
+
+	return func(ctx context.Context, t feature.T) {
+		if _, err := manifest.InstallYamlFS(ctx, yaml, cfg); err != nil {
+			t.Fatal(err)
+		}
+	}
+}
+
+func WithTriggerConsumerGroupIDTemplate(template string) manifest.CfgFn {
```
Comment (matzew) [inline test/rekt/resources/configmap/configkafkafeatures/configKafkaFeatures.go:44]: :+1:

**Review by matzew — APPROVED** [review]
/lgtm
/approve

/retest

**Comment by Cali0707** [comment]
Looks like that may have been a real failure

/cc @matzew

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**